python/test/unit/language/test_block_pointer.py

```diff
@@ -45,7 +48,7 @@ def test_block_copy(dtypes_str, n, padding_option, device):
     block_copy_kernel[grid](a_ptr=a, b_ptr=b, N=n, BLOCK_SIZE=64, padding_option=padding_option)
     a.to(dst_dtype)
     assert torch.all(a[0:n // 2] == b[0:n // 2])
-    if padding_option == "zero":
+    if padding_option is None or padding_option == "zero":
```

Comment:
default padding is not zero it is undefined

Reply:
oh I see. should this be documented explicitly then?

Reply:
probably, I thought it was but I cannot find it.